craft_platforms/_platforms.py

```diff
@@ -31,9 +31,14 @@
         "build-for": Annotated[Sequence[str], annotated_types.Len(1)],
     },
 )
+"""The architectures where a platform builds and where the resulting artifact runs."""
```

Comment:
```suggestion
"""The platforms where a platform builds and where the resulting artifact runs."""
```

Since this could be a multi-base, right?

Reply:
True, updated